### PR TITLE
[12.0] [MIG] account invoice update wizard

### PR DESCRIPTION
**Pull request by i-vyshnevska**
migrate account_invoice_update_wizard to 12.0

### CODE REVIEW

File: account_invoice_update_wizard/__manifest__.py [hunk 1]
```diff
@@ -1,34 +1,19 @@
 # -*- coding: utf-8 -*-
```
Comment (jcoux) [inline account_invoice_update_wizard/__manifest__.py]: No need to specifiy encoding with python 3:
```suggestion
```

File: account_invoice_update_wizard/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import account_invoice
```
Comment (jcoux) [inline account_invoice_update_wizard/models/__init__.py]:
```suggestion
from . import account_invoice

```

File: account_invoice_update_wizard/__manifest__.py [hunk 3]
```diff
@@ -1,34 +1,19 @@
...
-# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
-# Copyright 2018 Camptocamp
+# Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2019 Camptocamp
```
Comment (jcoux) [inline account_invoice_update_wizard/__manifest__.py]:
```suggestion
# Copyright 2018-2019 Camptocamp
```

File: account_invoice_update_wizard/tests/test_account_invoice_update_wizard.py [hunk 4]
```diff
@@ -1,53 +1,51 @@
-# -*- coding: utf-8 -*-
-# Copyright 2018 Camptocamp
+# Copyright 2019 Camptocamp
```
Comment (jcoux) [inline account_invoice_update_wizard/tests/test_account_invoice_update_wizard.py]:
```suggestion
# Copyright 2018-2019 Camptocamp
```

File: account_invoice_update_wizard/wizard/account_invoice_update.py [hunk 5]
```diff
@@ -1,6 +1,5 @@
-# -*- coding: utf-8 -*-
-# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
-# Copyright 2018 Camptocamp
+# Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2019 Camptocamp
```
Comment (jcoux) [inline account_invoice_update_wizard/wizard/account_invoice_update.py]:
```suggestion
# Copyright 2018-2019 Camptocamp
```

File: account_invoice_update_wizard/views/account_invoice.xml [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+<record id="invoice_supplier_form" model="ir.ui.view">
```
Comment (jcoux) [inline account_invoice_update_wizard/views/account_invoice.xml:9]: Bad indentation.

File: account_invoice_update_wizard/wizard/account_invoice_update_view.xml [hunk 7]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+<record id="account_invoice_update_form" model="ir.ui.view">
```
Comment (jcoux) [inline account_invoice_update_wizard/wizard/account_invoice_update_view.xml]: Bad indentation.

**Review by jcoux — CHANGES_REQUESTED** [review]

File: account_invoice_update_wizard/views/account_invoice.xml (hunk 6 above)
Comment (sebalix) [reply account_invoice_update_wizard/views/account_invoice.xml:9]: It is not important I think, and the code was like that before (not part of the migration). Often the indentation is done as if `<odoo>` tag was not there to gain line length.

**Review by jcoux — APPROVED** [review]
Thanks for changes

**Review by bealdav — APPROVED** [review]
Just tested ok.
Thanks

**Pull request merged**